plugins/inputs/elasticsearch/elasticsearch.go

```diff
@@ -17,71 +16,15 @@ import (
 	"strings"
 )
 
-// mask for masking username/password from error messages
-var mask = regexp.MustCompile(`https?:\/\/\S+:\S+@`)
```

Comment:
I noticed those changes when I did the merge. It was only a couple of lines, so I'll just include them in my next commit.